uwblueprint/feeding-canadian-kids
# 53 set password UI

## Zenhub ticket link

[Onboarding: Set Password UI](https://app.zenhub.com/workspaces/fck-627480c10d8974001872b0ef/issues/gh/uwblueprint/feeding-canadian-kids/53)



## Implementation description
* Set password UI
* Error handling for fields
* Mobile Responsiveness 



## Steps to test
1. Enter a password, and another different password that doesn't match for "Confirm your password"
2. Click the 'Confrim' button and verify that a relevant error message appears
3. Enter matching passwords that are less than 8 characters long
4. Click the 'Confirm' button and verify that a relevant error message appears
5. Enter matching passwords that are 8 characters or longer
6. Click the 'Confirm' button and verify that all error messages go away



## What should reviewers focus on?
* The UI matches the ticket outlines and UI from the design in the figma. https://www.figma.com/file/wrb9DiczKUX9ux1HWPmeDO/FCK---Login%2C-Sign-Up-%26-Forget-Password?node-id=46%3A92&t=7HXB9xTIJvN9eI56-4


## Checklist
